bin/lang-js/src/function.ts

```diff
@@ -169,10 +177,17 @@ export async function executeFunction(kind: FunctionKind, request: Request) {
   console.log(JSON.stringify(result));
 }
 
+function timer(minutes: number): Promise<never> {
+  return new Promise((_, reject) => {
+    setTimeout(() => reject(new Error(`function timed out after ${minutes} minutes`)), minutes * 60 * 1000);
```

Comment:
`setTimeout` relies on milliseconds. You can provide fractional values for the `minutes` value, if desired.